cpp/include/raft/distance/detail/pairwise_distance_cutlass_base.cuh

```diff
@@ -59,26 +62,29 @@ template <typename DataT,
           typename IdxT,
           int VecLen,
           typename FinalLambda,
-          typename DistanceFn,
+          typename OpT,
           bool isRowMajor>
-void cutlassDistanceKernel(const DataT* x,
-                           const DataT* y,
-                           const DataT* xn,
-                           const DataT* yn,
-                           IdxT m,
-                           IdxT n,
-                           IdxT k,
-                           IdxT lda,
-                           IdxT ldb,
-                           IdxT ldd,
-                           OutT* dOutput,
-                           FinalLambda fin_op,
-                           DistanceFn dist_op,
-                           cudaStream_t stream)
+typename std::enable_if<ops::has_cutlass_op<OpT>::value>::type cutlassDistanceKernel(
```

Comment:
```suggestion
std::enable_if_t<ops::has_cutlass_op<OpT>::value> cutlassDistanceKernel(
```

Reply:
Great tip! Thanks! This is a lot less ugly.